web/client/components/map/openlayers/Map.jsx

```diff
@@ -66,7 +66,10 @@ class OpenlayersMap extends React.Component {
         wpsBounds: PropTypes.object,
         onWarning: PropTypes.func,
         maxExtent: PropTypes.array,
-        limits: PropTypes.object
+        limits: PropTypes.object,
+        onMouseOut: PropTypes.func,
+        mousePosition: PropTypes.bool,
+        isMouseMoveOverTools: PropTypes.bool
```

Comment:
Not needed anymore, I think you can remove it

Reply:
Done